msb/fusionlog/TimeSeriesLogger.py

```diff
@@ -5,35 +5,51 @@
 import uptime
 from random import random
 
+from msb.fusionlog.config import FusionlogConf
+
+
 class TimeSeriesLogger:
-    def __init__(self, topic, config, msb_sn=""):
-        self._config = config
+    def __init__(self, topic : str, config: FusionlogConf, msb_sn: str = ""):
+        self.config = config
         if msb_sn:
             self.msb_sn = msb_sn
         else:
-            self.msb_sn = self._config.serialnumber
```

Comment:
This could be removed and the serial number in the config can be used always.

Reply:
removed it, wasn't used anyway